shared/teams/add-members-wizard/add-from-where.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import * as Styles from '../../styles'
+import * as Container from '../../util/container'
+import * as Constants from '../../constants/teams'
+import * as Types from '../../constants/types/teams'
+import {ModalTitle} from '../common'
+
+type Props = {
+  newTeam?: boolean
+  onContinue: (where: 'keybase' | 'email' | 'contacts' | 'phone') => void
+  teamID: Types.TeamID
+}
+
+const AddFromWhere = ({newTeam, onContinue, teamID}: Props) => {
+  const {teamname} = Container.useSelector(s => Constants.getTeamMeta(s, teamID))
```

Comment:
Not that I expect the description to be changing while this modal is open, but it seems like generally a good practice to be as specific as possible with `useSelector` to minimize rerendering

Reply:
that's fair